src/actions/LoginActions.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+import request from 'superagent'
```

Comment:
which should we use superagent or the native node request library? any argument for superagent?